xbmc/pictures/PictureInfoTag.cpp

```diff
@@ -1033,7 +1033,7 @@ void CPictureInfoTag::SetInfo(int info, const std::string& value)
     }
   case SLIDE_EXIF_DATE_TIME:
     {
-      strcpy(m_exifInfo.DateTime, value.c_str());
+      strncpy(m_exifInfo.DateTime, value.c_str(), sizeof(m_exifInfo.DateTime) - 1);
```

Comment:
not really needed, because of `memset(&m_exifInfo, 0, sizeof(m_exifInfo))` (https://github.com/xbmc/xbmc/blob/4a913d0/xbmc/pictures/PictureInfoTag.cpp#L33)
If I still should add it just tell me.